drv/transceivers-server/src/udp.rs

```diff
@@ -297,8 +297,11 @@ impl ServerImpl {
                 let (num_status_bytes, result) = self.get_status(mask, out);
                 ringbuf_entry!(Trace::OperationNoFailResult(result));
                 let success = ModuleId::from(result.success());
-                let (final_payload_len, errored_modules) =
-                    self.handle_errors(modules, result, num_status_bytes, out);
+                let (final_payload_len, errored_modules) = self.handle_errors(
```

Comment:
I think this is now wrong – `handle_errors` returns the number of error bytes, so you need to add that to `num_status_bytes` to get `final_payload_len`